.github/workflows/build.yml

```diff
@@ -0,0 +1,47 @@
+name: Build & Test
+on:
+  push:
+    branches:
+      - master
+      - release-*
+  pull_request:
+env:
+  DOTNET_NOLOGO: true
+jobs:
+  build:
+    name: ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ windows-2019, ubuntu-18.04 ]
+      fail-fast: false
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Setup .NET SDK
+        uses: actions/setup-dotnet@v1
+        with:
+          dotnet-version: 5.0.x
+      - name: Setup .NET Core 3.1 runtime
+        uses: actions/setup-dotnet@v1
+        with:
+          dotnet-version: 3.1.x
+      - name: Restore
+        run: dotnet restore src/NServiceBus.sln --disable-parallel          
+      - name: Build
+        run: dotnet build src/NServiceBus.sln --configuration Release --no-restore
+      - name: Upload packages
+        if: matrix.os == 'windows-2019'
+        uses: actions/upload-artifact@v2
+        with:
+          name: NuGet packages
+          path: nugets/
+          retention-days: 7
+      - name: Run Windows tests
+        if: matrix.os == 'windows-2019'
+        run: dotnet test src/NServiceBus.sln --configuration Release --no-build --logger "GitHubActions;report-warnings=false"
```

Comment:
Out of interest, why `--logger "GitHubActions;report-warnings=false"`?

Reply:
If you look at info on the GitHubActions logger we added, if you don't do `report-warnings=false` it adds an annotation for skipped tests, meaning that a skipped test (which is no big deal and happens all the time) gets mixed in with actual test failures, creating unnecessary noise.